tests/unit/dataactvalidator/test_c11_cross_file.py

```diff
@@ -53,6 +53,11 @@ def test_success(database):
 
     assert number_of_errors(_FILE, database, models=[af, ap]) == 0
 
```

Comment:
Yes, there is already a test for `parent_award_id=None` AwardFinancialFactory and a random parent_award_id in AwardProcurementFactory.